uwazi/entity-page/src/lib/Attestation.svelte

```diff
@@ -0,0 +1,294 @@
+    let rect = attrTitle.getBoundingClientRect();
+    altModalPos.x = rect.left + 10;
+    altModalPos.y = rect.top + 10;
+    showAltModal = true;
+  }
+
+  function downloadClick() {
+    if (data.attestation.encrypted) {
+      return;
+    }
+    downloadDialog.showModal();
+  }
+
+  function isLargeData(value) {
+    return value.toString() === "[object Object]";
+  }
+
+  function trimLarge(value) {
+    let max = 100; // Long enough to be too wide and trigger CSS ellipsis
+    let string = JSON.stringify(value);
+    return string.length > max ? string.substring(0, max - 3) + "..." : string;
+  }
+
+  $: largeData = isLargeData(data.attestation.value);
+</script>
+
+<div id="container">
+  <div id="text">
+    <h2
+      class="attr"
+      bind:this={attrTitle}
+      on:click={attrTitleClick}
+      on:keydown={attrTitleClick}
+    >
+      {customTitle || data.attestation.attribute}
+    </h2>
+    <slot name="value">
+      {#if data.attestation.encrypted}
+        <span class="value encrypted">encrypted</span>
+      {:else if largeData}
+        <span class="value large">{trimLarge(data.attestation.value)}</span>
+      {:else}
+        <span class="value">{data.attestation.value}</span>
+      {/if}
+    </slot>
+  </div>
+  <div id="icons">
+    <svg
+      class="icon"
+      bind:this={attrIcon}
+      on:click={iconClick}
+      on:keydown={iconClick}
+      xmlns="http://www.w3.org/2000/svg"
+      viewBox="0 0 496 512"
+      ><!--! Font Awesome Pro 6.4.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license (Commercial License) Copyright 2023 Fonticons, Inc. --><path
+        d="M314.9 194.4v101.4h-28.3v120.5h-77.1V295.9h-28.3V194.4c0-4.4 1.6-8.2 4.6-11.3 3.1-3.1 6.9-4.7 11.3-4.7H299c4.1 0 7.8 1.6 11.1 4.7 3.1 3.2 4.8 6.9 4.8 11.3zm-101.5-63.7c0-23.3 11.5-35 34.5-35s34.5 11.7 34.5 35c0 23-11.5 34.5-34.5 34.5s-34.5-11.5-34.5-34.5zM247.6 8C389.4 8 496 118.1 496 256c0 147.1-118.5 248-248.4 248C113.6 504 0 394.5 0 256 0 123.1 104.7 8 247.6 8zm.8 44.7C130.2 52.7 44.7 150.6 44.7 256c0 109.8 91.2 202.8 203.7 202.8 103.2 0 202.8-81.1 202.8-202.8.1-113.8-90.2-203.3-202.8-203.3z"
+      />
+      <title>{signer}</title>
+    </svg>
+    <svg
+      class:disabled={data.attestation.encrypted}
+      on:click={downloadClick}
+      on:keydown={downloadClick}
+      class="icon"
+      xmlns="http://www.w3.org/2000/svg"
+      viewBox="0 0 512 512"
+      ><!--! Font Awesome Free 6.4.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license (Commercial License) Copyright 2023 Fonticons, Inc. --><path
+        d="M448 384H256c-35.3 0-64-28.7-64-64V64c0-35.3 28.7-64 64-64H396.1c12.7 0 24.9 5.1 33.9 14.1l67.9 67.9c9 9 14.1 21.2 14.1 33.9V320c0 35.3-28.7 64-64 64zM64 128h96v48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16H256c8.8 0 16-7.2 16-16V416h48v32c0 35.3-28.7 64-64 64H64c-35.3 0-64-28.7-64-64V192c0-35.3 28.7-64 64-64z"
+      /></svg
+    >
+  </div>
```

Comment:
Clicking on the "clone/copy" icon which replaced the download icon opens up this dialog, but isn't the intended use the opposite? We want to send data from the Hyperbee instance to the UWAZI db? I might just be out of the loop though, let me know if I'm missing something
![Screenshot 2023-07-17 at 3 07 10 PM](https://github.com/starlinglab/authenticated-attributes/assets/2441069/d9fefcfb-7ab9-4d86-9926-a60b8f7c76ee)